toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1726,6 +1726,14 @@ void MainWindow::defineActions() {
                          "");
   createMenuXsheetAction(MI_RemoveGlobalKeyframe, tr("Remove Multiple Keys"),
                          "");
+  createMenuXsheetAction(MI_FirstFrame, tr("First Frame"), "");
+  createMenuXsheetAction(MI_LastFrame, tr("Last Frame"), "");
+  createMenuXsheetAction(MI_PrevFrame, tr("Previous Frame"), "Shift+A");
+  createMenuXsheetAction(MI_NextFrame, tr("Next Frame"), "Shift+S");
+  createMenuXsheetAction(MI_NextDrawing, tr("Next Drawing"), "Shift+X");
+  createMenuXsheetAction(MI_PrevDrawing, tr("Prev Drawing"), "Shift+Z");
+  createMenuXsheetAction(MI_NextStep, tr("Next Step"), "");
+  createMenuXsheetAction(MI_PrevStep, tr("Prev Step"), "");
 
```

Comment:
Here you changed types of these commands from PlaybackCommandType to MenuXsheetCommandType. 
Is it only because you would like to include these commands into the menu bar customization popup?
If so, how about to make the menu bar customization popup to browse the playback commands additionally,  instead of changing command types? Because we already have too many Xsheet commands...